src/test/request.test.js

```diff
@@ -0,0 +1,69 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+import server from '..';
+import { newCompanyUser, createCompanyFacility } from './dummies';
+import { AuthController, RequestController } from '../controllers';
+import { RequestService } from '../services';
+
+const { companySignUp, userSignup } = AuthController;
+
+chai.use(chaiHttp);
+chai.use(sinonChai);
+
+const [companyAdmin] = createCompanyFacility;
+
+describe('Request route endpoints', () => {
+  let adminToken;
+  let companyAdminResponse;
+  // let companyUserResponse;
+
+  before(async () => {
+    const reqCompany = { body: { ...companyAdmin, email: 'baystef@slack.com', companyName: 'paystack' } };
+
+    const res = {
+      status() {
+        return this;
+      },
+      cookie() {
+        return this;
+      },
+      json(obj) {
+        return obj;
+      }
+    };
+
+    companyAdminResponse = await companySignUp(reqCompany, res);
+    const { data: { signupToken } } = companyAdminResponse;
+    const reqUser = { body: { ...newCompanyUser, email: 'steve@google.com', signupToken, roleId: 5 } };
+    const companyUserResponse = await userSignup(reqUser, res);
+    adminToken = companyUserResponse.data.token;
+  });
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('GET api/users/requests', () => {
+    it('should return 404 for user with no request yet', async () => {
```

Comment:
Please include test case for 200. You can create a request in the db for the user to do this. Ensure to clean up the database after the test for a clean database

Reply:
Thanks,  I was actually trying to fake it with Sinon, but it didn't work. I'll implement your suggestion right now.